presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/optimization/JdbcFilterToSqlTranslator.java

```diff
@@ -160,6 +154,22 @@ public JdbcFilterToSqlTranslator(FunctionMetadataManager functionMetadataManager
         return untranslated(specialForm, translatedExpressions);
     }
 
+    public static List<String> mergeSqlBodies(List<JdbcExpression> jdbcExpressions)
+    {
+        return jdbcExpressions.stream()
+                .map(JdbcExpression::getExpression)
+                .map(sql -> '(' + sql + ')')
+                .collect(toImmutableList());
+    }
+
+    public static List<ConstantExpression> mergeVariableBindings(List<JdbcExpression> jdbcExpressions)
+    {
+        return jdbcExpressions.stream()
+                .map(JdbcExpression::getBoundConstantValues)
+                .flatMap(List::stream)
+                .collect(toImmutableList());
+    }
```

Comment:
It's good to extract the two helpers; but once extracted out, they don't belong to this class anymore. Move them to JdbcTranslationUtil